### PR TITLE
Deploy script

### PR DESCRIPTION
**Pull request by DieMyst**
- Updates deploy script
- CLI can decode base64 tendermint key
- A small script to decode ethereum keystore to a private key

### CODE REVIEW

File: bootstrap/unlock-keys.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+// the script gets private key from ethereum keystore
+var keythereum = require("keythereum");
```
Comment (alari) [inline bootstrap/unlock-keys.js]: What is the purpose for this script? Why do we need it at all? In what scenario should it be used?

File: cli/src/register.rs [hunk 2]
```diff
@@ -39,6 +40,7 @@ const ETH_URL: &str = "eth_url";
 const PASSWORD: &str = "password";
 const SECRET_KEY: &str = "secret_key";
 const WAIT_SYNCING: &str = "wait_syncing";
+const BASE64_TENDERMINT_KEY: &str = "base64";
```
Comment (alari) [inline cli/src/register.rs]: Is it a good value for the Tendermint key argument?

File: tools/run/compose.sh [hunk 3]
```diff
@@ -1,15 +1,67 @@
...
 
    Linux)
      export DOCKER_IP=$(ifconfig docker0 | grep 'inet ' | awk '{print $2}' | grep -Po "[0-9\.]+")
      export HOST_IP=$(ip route get 8.8.8.8 | grep -Po "(?<=src )[0-9\.]+")
-     docker-compose up -dV
      ;;
 esac
+
+# running parity and swarm containers
+docker-compose -f parity.yml up -d
```
Comment (alari) [inline tools/run/compose.sh]: Could we have a single docker-compose file with several services inside?

File: bootstrap/unlock-keys.js (hunk 1 above)
Comment (DieMyst) [reply bootstrap/unlock-keys.js]: it is needed to decode keystore to the private key until it will be supported in CLI

Comment (alari) [reply bootstrap/unlock-keys.js]: You should describe it properly here it comments and/or in readme

**Pull request merged**